pkg/plugins/golang/options.go

```diff
@@ -0,0 +1,256 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Before the options were in `pkg/model/resource/options.go`. Why move out for plugins/golang?

Reply:
`Options` was created before the plugin system, so it being in the `pkg/model` dir made sense. However, with the plugin system `Options` has proven to be related only to the `go` plugin and not to the library, and therefore it makes more sense to be palced where it is now.